guides/common/modules/ref_ipv6-and-ipv4-requirements.adoc

```diff
@@ -0,0 +1,43 @@
+[id="ipv6-and-ipv4-requirements"]
+= IPv6 and IPv4 requirements
+
+You can install {Project} in an IPv4 network or in an IPv6 network.
+ifdef::satellite[]
+Dual-stack {Project} installation that uses both IPv4 and IPv6 is not supported.
+endif::[]
+
+The following requirements apply to installations in an IPv4 network:
+
+* Ensure an IPv6 loopback is configured on the base system.
+The loopback is typically configured by default.
+Do not disable it.
+* Do not disable IPv6 in kernel by adding the `ipv6.disable=1` kernel parameter.
+
+The following requirements apply to installations in an IPv6 network:
+
+ifdef::satellite[]
+* Deploy an external HTTP proxy server that supports both IPv4 and IPv6.
+This is required because Red Hat Content Delivery Network distributes content only over IPv4 networks, therefore you must use this HTTP proxy to pull content into the {Project} on your IPv6 network.
+You must configure {Project} to use this dual-stack (supporting both IPv4 and IPv6) HTTP proxy as the default HTTP proxy.
+endif::[]
+ifndef::foreman-deb,foreman-el,satellite[]
+* If you rely on content from IPv4-only networks, use an external dual-stack HTTP proxy.
+Configure {Project} to use this dual-stack (supporting both IPv4 and IPv6) HTTP proxy as the default HTTP proxy.
+endif::[]
+* {Project} does not support configuring an HTTP proxy using a direct IPv6 address.
+Instead, configure the HTTP proxy with a FQDN that resolves to the IPv6 address.
+Using an IPv6 address as the HTTP proxy URL causes it to fail.
+
+If you also intend to provision hosts in an IPv6 network, consider the following requirements:
```

Comment:
```suggestion
The following requirements apply to provisioning hosts in an IPv6 network:
```

suggestion (non-blocking): Align with lines 9 and 16.

Reply:
Applied, except that I kept "If you intend to [...]" at the beginning. That's import so that users know they can skip this if they don't meet the condition.